src/condition/ssrf_detector.hpp

```diff
@@ -13,7 +13,7 @@ namespace ddwaf {
 
 class ssrf_detector : public base_impl<ssrf_detector> {
 public:
-    static constexpr unsigned version = 1;
+    static constexpr unsigned version = 2;
```

Comment:
We're not doing `ssrf_detector@v2`, right?

Reply:
I was overzealous and wanted to do everything here, but I changed this before you commented.